src/components/Sum.jsx

```diff
@@ -1 +1,11 @@
-// export function Sum() {}
+import React from 'react';
+
+function Sum({ a = 0, b = 0 }) {
```

Comment:
```suggestion
export function Sum({ a = 0, b = 0 }) {
```
so you can pass the tests 😉